tests/integration/seccomp.bats

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bats
+
+load helpers
+
+SECCOMP_TESTS_BINDIR="$(mktemp -d "$BATS_TMPDIR/seccomp-bins.XXXXXX")"
+SECCOMP_TESTS=(
+	"seccomp_syscall_test1"
+)
+
+function setup() {
+	teardown_busybox
+	setup_busybox
+
+	# Compile the test cases...
+	for test_name in "${SECCOMP_TESTS[@]}"; do
+		gcc -static -o "$SECCOMP_TESTS_BINDIR/$test_name" "$TESTDATA/$test_name.c"
```

Comment:
note that `setup()` is run for _every_ `@test`. Once the second test case is added, this logic (of compiling all *.c files for every test) will look weird.

Reply:
Yeah, I'll move compilation to the `@test`.